nengo_spa/vocab.py

```diff
@@ -111,6 +112,8 @@ def __iter__(self):
 
     def __getitem__(self, key):
         """Return the semantic pointer with the requested name."""
+        if key == '__tracebackhide__':
```

Comment:
Is this used only for testing? That's what I got from consulting StackOverflow. If it is, what test is it specifically used in? You may want to a comment, since it's an unusual piece of code that may merit explanation.